content/asset-tracking/using-the-sdks.textile

```diff
@@ -680,3 +680,4 @@ h2(#see-also). See also
 * "Using the example apps":/asset-tracking/example-apps
 * "Asset Tracking Android training workshop project":https://github.com/ably/asset-tracking-android-training
 * "Reference documentation generated from SDK source code":https://sdk.ably.com/builds/ably/ably-asset-tracking-android/main/dokka/index.html
+* "Reference documentation for Ably Asset Tracking SDKs for Swift":https://sdk.ably.com/builds/ably/ably-asset-tracking-swift/main/jazzy/
```

Comment:
Good, but can we also update the line above to be like:

"Ably Asset Tracking Android SDK Reference - generated from source code":https://sdk.ably.com/builds/ably/ably-asset-tracking-android/main/dokka/index.html

and then change the Swift line to be consistent with the Android line